app/dashboard/templates/ens/base.html

```diff
@@ -5,32 +5,17 @@
     <title>{% trans "ENS Subdomains | Gitcoin" %}</title>
     {% include 'shared/cards_pic.html' with title='Gitcoin | _you_.gitcoin.eth ENS Subdomains'  card_desc='*NEW* Register `_you_.gitcoin.eth` with one click for free' avatar_url="https://s.gitcoin.co/static/v2/images/ens_gitcoin.png"  %}
     {% include 'shared/head.html' %}
-    <meta charset="utf-8" />
-    <meta name="title" content="ENS Subdomains | Gitcoin">
-    <meta name="description" content="Create and manage your ENS subdomain">
-    <meta name=viewport content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=no">
-    <meta name="theme-color" content="#ffffff">
-    <!--[if lte IE 8]><script src="{% static "onepager/js/html5shiv.js" %}"></script><![endif]-->
-    <link rel="stylesheet" href="{% static "v2/css/bootstrap.min.css" %}" crossorigin="anonymous">
+
     <link rel="stylesheet" href="{% static "v2/css/kudos/styles.css" %}" />
     <link rel="stylesheet" href="{% static "onepager/css/main.css" %}">
-    <link rel="stylesheet" href="{% static "v2/css/lib/typography.css" %}">
-    <link rel="stylesheet" href="{% static "v2/css/gitcoin.css" %}">
-    <link rel="stylesheet" href="{% static "onepager/css/main.css" %}">
     <link rel="stylesheet" href="{% static "v2/css/box_redeem.css" %}">
-    <link rel="stylesheet" href="{% static "v2/css/rain.css" %}">
-    <link rel="stylesheet" href="{% static "v2/css/forms/button.css" %}">
-    <link rel="stylesheet" href="{% static "v2/css/external_bounties/checkboxes.css" %}">
-    <link rel="stylesheet" href="{% static "v2/css/jquery.select2.min.css" %}">
-    <link rel="stylesheet" href="{% static "cookielaw/css/cookielaw.css" %}">
 
     <script src="{% static "v2/js/lib/jquery.js" %}"></script>
     <script src="{% static "v2/js/metamask-approval.js" %}"></script>
     <script src="{% static "v2/js/work_with_gitcoin.js" %}"></script>
     <!--[if lte IE 9]><link rel="stylesheet" href="{% static "onepager/css/ie9.css" %}"><![endif]-->
     <!--[if lte IE 8]><link rel="stylesheet" href="{% static "onepager/css/ie8.css" %}"><![endif]-->
     <noscript><link rel="stylesheet" href="{% static "onepager/css/noscript.css" %}"></noscript>
-    {% include 'shared/favicon.html' %}
```

Comment:
shouldn't we leave this in ?

Reply:
[On line 7](https://github.com/gitcoinco/web/blob/e4ac43b3e19983941c2ed0595c7971598b37c475/app/dashboard/templates/ens/base.html#L7) we include the `shared/head.html`. The `shared/head.html` includes for its part the [shared/favicon.html](https://github.com/gitcoinco/web/blob/a6d073c3c8a3be999269e1ae0b390b13ff2cb814/app/retail/templates/shared/head.html#L75). So we don't need it twice.